src/ApolloHooksSubscription.re

```diff
@@ -0,0 +1,77 @@
+module type Config = {
```

Comment:
Its already defined in ApolloHooksTypes.re (I left comment there about use this type from apollo-reason)

Reply:
Yes good one!